lib/segment/src/vector_storage/query_scorer/mod.rs

```diff
@@ -28,7 +28,7 @@ pub fn score_multi<TMetric: Metric>(
         sum += multidense_b
             .iter()
             .map(|dense_b| TMetric::similarity(dense_a, dense_b))
-            .fold(0.0, |a, b| if a > b { a } else { b });
+            .fold(ScoreType::NEG_INFINITY, |a, b| if a > b { a } else { b });
```

Comment:
Is this a fix to handle negative scores OR for empty multi dense vectors?

Reply:
Handling negative scores. An empty multivectors score isn't defined and will be trigger validation errors in future from api's